Generator/Program.cs

```diff
@@ -7,12 +7,12 @@ class Program
     {
         static void Main(string[] args)
         {
-            var generator = new CodeGenerator(@"..\..\..\..\glTF\specification\2.0\schema\glTF.schema.json");
+            var generator = new CodeGenerator(@"..\..\..\..\..\glTF\specification\2.0\schema\glTF.schema.json");
```

Comment:
This doesn't seem right. The expectation is that the glTF repo is at the same level as the glTF-CSharp-Loader. This change breaks that.